STYLE_GUIDE.md

```diff
@@ -37,4 +37,9 @@ BAR = [
 # pylint: disable=protected-access
 foo._protected_member
 # pylint: enable=protected-access
-```
\ No newline at end of file
+```
+
+
+##### TensorFLow Conventions
```

Comment:
Nit: four '#', instead of five '#'

Reply:
I had put it as a subsection under Python since TensorFlow conventions are only listed under Python, and C++ sections are same. Changed to four '#'.